lib/taskana-history-plugin/.gitignore

```diff
@@ -0,0 +1,29 @@
+*/target/
```

Comment:
again, one .gitignore file should suffice

Reply:
same as above

Reply:
done